authz/sdk_end2end_test.go

```diff
@@ -249,11 +251,13 @@ func TestSDKEnd2End(t *testing.T) {
 					}
 				]
 			}`,
-			wantStatusCode: codes.PermissionDenied,
-			wantErr:        "unauthorized RPC request rejected",
-		},
-	}
-	for name, test := range tests {
+		wantStatusCode: codes.PermissionDenied,
+		wantErr:        "unauthorized RPC request rejected",
+	},
+}
+
+func TestSDKStaticPolicyEnd2End(t *testing.T) {
```

Comment:
Please see our other packages for how we write tests in grpc-go to ensure leak checking is performed after each test is run:

https://github.com/grpc/grpc-go/blob/78d3aa8b3ed1b59bf84db4242ac7c316e8943797/grpc_test.go#L27-L33

https://github.com/grpc/grpc-go/blob/78d3aa8b3ed1b59bf84db4242ac7c316e8943797/clientconn_test.go#L44

Reply:
Done.